src/backend/joanie/remote_endpoints_urls.py

```diff
@@ -0,0 +1,15 @@
+"""
+API routes exposed for server to server. Requires a specific token to request.
+"""
+from django.conf import settings
+from django.urls import re_path
+
+from joanie.core.api.remote_endpoints import enrollments_and_orders_on_course_runs
+
+urlpatterns = [
+    re_path(
+        rf"api/{settings.API_VERSION}/enrollments_and_orders_on_course_runs/?$",
```

Comment:
It's better to use `-` here.
```suggestion
        rf"api/{settings.API_VERSION}/enrollments-and-orders-on-course-runs/?$",
```
Also, do we need to use a regex here ?